iml-wasm-components/iml-action-dropdown/src/lib.rs

```diff
@@ -62,6 +62,14 @@ fn show_class(model: &Model) -> &str {
     }
 }
 
+fn composite_ids_to_query_string(compositeIds: Vec<Record>) -> String {
```

Comment:
Since you've already created separate modules besides lib, you should organize new code into modules.

Reply:
Sure, for this I could put it in `api-transforms.rs`?

Reply:
maybe just in locks.rs for now? It's related to locks.